code/modules/language/zombie.dm

```diff
@@ -0,0 +1,14 @@
+/datum/language/zombie
+	name = "Zombie"
+	desc = "The common tongue of the zombies."
+	speech_verb = "moans"
+	ask_verb = "moans"
+	exclaim_verb = "moans"
+	sing_verb = "moans melodically"
+	key = "z"
+	syllables = list("BRAAAAIIINS", "oRr" ,"ggrr" ,"orR" ,"RRROO", "aaHHh" ,"grrr" ,"RRaa" ,"gRRr" ,"GrR" ,"ara" ,"raaAAA" ,"mMmRrR", "ssRR",\
+	"oOORRrr" ,"GrRRR" ,"uRRRRR" ,"ssRr" ,"OrR" ,"RrrOo" ,"AhhhH" ,"ara" ,"UuUrRRrrR" ,"RrrAA" ,"RaAAAaaA" ,"Rrrrrr" ,"araa" ,"RAa" ,"gRRrRr" ,"araa" ,"RrRRAa", "Ooooo",\
+	"nuurg", "gGGgRR", "MmmRR")
```

Comment:
couple issus here;
dont use \ at the end of lists
missing trailing comma
dupe syllables (ex: araa)
id rather not have ara because uhh yea
Oooo is for ghosts
The random capitilization is frankly annoying tbh
if you want I can give you a list of the syllables i woud use

Reply:
Yes, i'd like a list of syllabes